README.md

```diff
@@ -1 +1,17 @@
-Keep your branches clean and nice. Use flake8 and tests. Good luck!
\ No newline at end of file
+# .env:
+
+database name, host, port, passsword and user to connect 
+
+- PG_HOST: postgresql host
```

Comment:
how do i know this creds? you set docker pg up yourself with your script.
maybe you could set up .env file in that script too?